master_discovery_fkie/src/master_discovery_fkie/master_discovery.py

```diff
@@ -341,6 +342,12 @@ def __retrieve_masterinfo(self):
                     if float(timestamp) != 0:
                         self.masteruri = masteruri
                         self.mastername = mastername
+                        if masteruri is None:
+                            msg = "DiscoveredMaster[%s] Returned a Master URI of NONE" % self.monitoruri
+                            rospy.logwarn(msg)
+                        if mastername is None:
+                            msg = "DiscoveredMaster[%s] Returned a Master Name of NONE" % self.monitoruri
```

Comment:
oh actually no this is correct. The monitor URI is meant to identify the DiscoveredMaster instance. The self.mastername is none in this case.